public/hackathon/index.html

```
@@ -1332,37 +1244,57 @@ <h2 id="start-title">Manual setup: SDK step by step</h2>
-const connectedAccount = response.connectedAccount;
+let connectedAccount = response.connectedAccount;
 
+// Do not call tools until the user finishes OAuth and status is ACTIVE.
 if (connectedAccount?.status !== ConnectorStatus.ACTIVE) {
   const linkResponse = await actions.getAuthorizationLink({
     connectionName,
     identifier: 'user_123',
   });
   console.log('Authorize Gmail:', linkResponse.link);
+  console.log('Press Enter after authorizing Gmail...');
+  await new Promise((resolve) => {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)